server/tests/routes.test.js

```diff
@@ -70,6 +71,28 @@ describe("API POST Data", () => {
       res.body.every((item) => testBarcodes.includes(item["barcode"]))
     ).toBe(true);
   });
+  it("should add a new order", async () => {
+    const testUserId = TEST_USERS[0]["user-id"];
+    const res = await request(app).post("/api/order").send({
+      "merchant-id": "TEST_MERCHANT",
+      "order-id": "TEST_ORDER",
+      "user-id": "TEST_USER",
+    });
+    expect(res.statusCode).toEqual(CONSTANTS.HTTP_SUCCESS);
```

Comment:
does the api return anything other than success status e.g. order ID? should we check that too?